lxc/action.go

```diff
@@ -110,12 +112,31 @@ func (c *actionCmd) doAction(conf *config.Config, nameArg string) error {
 }
 
 func (c *actionCmd) run(conf *config.Config, args []string) error {
+	var names []string
 	if len(args) == 0 {
-		return errArgs
+		if !c.all {
+			return errArgs
+		}
+		d, err := conf.GetContainerServer(conf.DefaultRemote)
+		if err != nil {
+			return err
+		}
+		ctslist, err := d.GetContainers()
+		if err != nil {
+			return err
+		}
+		for _, ct := range ctslist {
+			names = append(names, ct.Name)
+		}
+	} else {
+		if c.all {
+			return fmt.Errorf("Both --all and container names given")
```

Comment:
Should use i18n.G to have the string translated

Reply:
Fixed, thanks.